src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/StructuralVariationProgramGroup.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramGroup;
+
+public final class StructuralVariationProgramGroup implements CommandLineProgramGroup {
+
+    @Override
+    public String getName() {
+        return "Spark tools for structural variation analysis";
```

Comment:
either add Spark to the class name or remove Spark from this string and the description string.